Tasks/Sushchynskiy_Tasks/ClassWorks/ClassWork2.py

```diff
@@ -0,0 +1,45 @@
+a = "thirteen five thirteen five seventeen fourteen fifteen one thirteen fourteen twenty"
+b = a.split(" ")
+
+# из члов -> в числа
+
+c = {"one": 1, "two": 2, "three": 3, "four": 4, "five": 5, "six": 6, "seven": 7, "eight": 8, "nine": 9, "ten": 10, "eleven": 11, 
+"twelve": 12, "thirteen": 13, "fourteen": 14, "fifteen": 15, "sixteen": 16, "seventeen": 17, "eighteen": 18, "nineteen": 19, "twenty": 20}
+
+a1 = []
+for i in b:
+    for j in c:
+        if i == j:
+            a1.append(c[j])
```

Comment:
why not just use dictionary indexing like
```suggestion
for i in b:
    a1.append(c[i])
```